### DISCUSSION TITLE
Refetch from another query

### DISCUSSION THREAD

**Discussion by hyusetiawan** [gqty-dev/gqty#1603] [Q&A]
suppose I have a list of `Chat` being fetched through the following query: `session.user.chats.map((chat =>`, I am listing the chats that the currently logged in user has.
Then I have a button called a new chat in a separate component that creates a new chat, the question is how do I update the list or trigger the `session.user.chats` to refetch?

**Comment by vicary** [comment]
```ts
const query = useQuery();

query.$refetch();
```

**Comment by hyusetiawan** [comment]
ah of course, but will it refetch ALL of the queries or specific queries
that might get affected?

**Reply by vicary** [reply]
Basically, only the selections from that very useQuery hook.

I have hacks that combine fetches if you mix the proxy objects up by sending children upwards. If you are not doing those naughty things, you should see clean and minimal fetches.
